src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/viewcontent/view/ViewContent.kt

```diff
@@ -0,0 +1,87 @@
+package org.wycliffeassociates.otter.jvm.app.ui.viewcontent.view
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.beans.property.Property
+import javafx.scene.layout.Priority
+import javafx.stage.Screen
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.common.data.model.Content
+import org.wycliffeassociates.otter.jvm.app.images.ImageLoader
+import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.ui.viewcontent.ViewContentStyles
+import org.wycliffeassociates.otter.jvm.app.ui.viewcontent.viewmodel.ViewContentViewModel
+import org.wycliffeassociates.otter.jvm.app.widgets.card.DefaultStyles
+import org.wycliffeassociates.otter.jvm.app.widgets.card.card
+import tornadofx.*
+
+
+class ViewContent : Fragment() {
```

Comment:
What is a ViewContent?

Reply:
it's a content grid. I didn't choose the name very well